frontend/src/components/common/Autocomplete.tsx

```diff
@@ -20,6 +22,9 @@ export function NamespacesAutocomplete() {
   const theme = useTheme();
   const filter = useTypedSelector(state => state.filter);
   const [namespaces, setNamespaces] = React.useState<Namespace[]>([]);
+  const queryParamDefaultObj = {'namespace': '', 'page': '0'};
```

Comment:
Do we need a page here? (and below)

Reply:
aah so this shouldn't be here but one thing in SimpleTable Component we must look for changes in the url namespace filter and whenever it changes we must reset the page to 0 because the namespace filtering will triger but if don't reset the page to 0 and if we our page was previously pointing to no 2 but because of the filter we don't have that much data we will show no data because of being on the wrong page

Reply:
@joaquimrocha this won't be required anymore created and used checkOutOfBoundUrlParam function which will handle this